lib/diagnostic.js

```diff
@@ -51,7 +93,11 @@ db.dropDatabase();
 // House Stark.
 
 // your code begins here
-
+db.house.deleteMany(
+  {
+    house: "Stark"
```

Comment:
Keys are case-sensitive.  This won't work since the original documents have `House` rather than `house` as keys.